src/walk.js

```diff
@@ -0,0 +1,13 @@
+import * as walk from 'acorn/dist/walk';
+import { DynamicImportKey } from './inject';
+
+export function inject(injectableWalk) {
+  return { ...injectableWalk,
+    base: {
+      ...injectableWalk.base,
+      [DynamicImportKey]: () => {},
+    } };
+}
```

Comment:
This will only work in node >= 8.

Reply:
Since the node_module points to a built version I didn't think that would matter, but I can change to `Object.assign` for people that use this directly.